modules/blocks/process.js

```diff
@@ -487,10 +487,10 @@ Process.prototype.loadBlocksFromPeer = function(peer, cb) {
 			if (!err) {
 				// Update last valid block
 				lastValidBlock = block;
-				library.logger.info(
-					['Block', block.id, 'loaded from:', peer.string].join(' '),
-					`height: ${block.height}`
-				);
+				library.logger.info('Block loaded', {
+					blockId: block.id,
+					peer: peer.string,
+				});
```

Comment:
Bunyan consider that first param must be `object` and additional message should be passed as `second argument`. How you override this behavior? 

https://github.com/trentm/node-bunyan#log-method-api